layers/meta-resin-raspberrypi/recipes-core/packagegroups/packagegroup-resin-connectivity.bbappend

```diff
@@ -1 +1 @@
-CONNECTIVITY_FIRMWARES_append_raspberrypi3 = " linux-firmware-brcm43430"
```

Comment:
NIT: "This file doesn't exist in barys anymore". Barys is just a runner. You mean in meta-raspberrypi.